gateway-service-factory/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 dependencies {
-  api("org.hypertrace.core.serviceframework:platform-grpc-service-framework:0.1.58")
+  api("org.hypertrace.core.serviceframework:platform-grpc-service-framework:0.1.59")
```

Comment:
can we bring this up to date with latest? Also guessing there's more framework libs that should be brought up to same version - at least metrics should be hiding somewhere, since I know we have them for GW.